src/backend/constants.ts

```diff
@@ -82,7 +82,9 @@ const weblateUrl = 'https://hosted.weblate.org/projects/heroic-games-launcher'
 const kofiPage = 'https://ko-fi.com/heroicgames'
 const patreonPage = 'https://www.patreon.com/heroicgameslauncher'
 const wineprefixFAQ = 'https://wiki.winehq.org/FAQ#Wineprefixes'
-
+const bottlesWineBin = fixAsarPath(
+  join(publicDir, 'bin', 'linux', 'bottles-wine')
+)
```

Comment:
would it be useful to add this path in the logs when Heroic starts? we are already printing Gogdl and Legendary

I've never used bottles so I don't know if it might be useful to print the path to bottles for when we need to help people with issues or if the `wineVersion` setting will include all we need

Reply:
Well, the script path isn't all too interesting